apps/earn-protocol/features/bridge/hooks/use-bridge-transaction.ts

```
@@ -0,0 +1,220 @@
+type BridgeTransactionInfoWithLzFeeUsd = BridgeTransactionInfo & {
+  metadata: { lzFeeUsd: string }
+}
+
+/**
+ * Return type for the useBridgeTransaction hook
+ */
+interface BridgeTransactionDetails {
+  /** Prepares the bridge transaction with optional override amount */
+  prepareTransaction: (overrideAmount?: string) => void
+  /** Executes the prepared bridge transaction */
+  executeBridgeTransaction: () => void | SendUserOperationWithEOA<unknown>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)